correlation_clustering/data_proc.py

```diff
@@ -0,0 +1,142 @@
+import numpy as np
+
+
+def build_graph(tsvfilename):
+    f = open(tsvfilename)
```

Comment:
tu ouvres un fichier mais tu le refermes pas. En gros généralement quand tu ouvres un fichier faut le refermer parce que sinon ça laisse un buffer ouvert et ça prend de la place, cf : https://docs.python.org/2/tutorial/inputoutput.html#methods-of-file-objects